src/lucky_migrator/create_table_statement.cr

```diff
@@ -144,4 +167,8 @@ class LuckyMigrator::CreateTableStatement
       " NOT NULL"
     end
   end
+
+  def references(table_name)
```

Comment:
And could you add a type restriction here: `table_name : String?` so it's clear what the options can be?